alvr/dashboard/src/launcher/mod.rs

```diff
@@ -0,0 +1,29 @@
+mod commands;
+
+use std::{sync::mpsc, thread, time::Duration};
+
+use crate::WorkerMsg;
+
+pub fn launcher_thread(rx1: mpsc::Receiver<WorkerMsg>) {
```

Comment:
Why is the launcher code being run in a separate thread?

Reply:
Oh, it seemed wrong to do it in the gui thread to me, but i guess nothing is happening in the ui at that time anyway.